### PR TITLE
Separate ossec.conf into agent.conf and manager.conf

### PR DESCRIPTION
**Pull request by Nicogp**
|Related issue|
|---|
|#7792|

## Description

Rename all references to ossec.conf file

## Tests

- Compilation without warnings in every supported platform
  - [x] Linux
  - [x] Windows
- [x] Source installation
- [x] Source upgrade
- [x] Review logs syntax and correct language

### CODE REVIEW

File: src/win32/setup-syscheck.c [hunk 1]
```diff
@@ -33,9 +33,9 @@ int main(int argc, char **argv)
         return (0);
     }
 
-    /* Check if OSSEC-HIDS was installed already */
+    /* Check if Agent was installed already */
     if (!fileexist(AGENTCONF)) {
-        printf("%s: OSSEC not installed yet. Exiting.\n", argv[0]);
+        printf("%s: Agent not installed yet. Exiting.\n", argv[0]);
```
Comment (TomasTurina) [inline src/win32/setup-syscheck.c:38]:
```suggestion
        printf("%s: WAZUH not installed yet. Exiting.\n", argv[0]);
```

File: src/config/config.c [hunk 2]
```diff
@@ -65,11 +65,27 @@ static int read_main_elements(const OS_XML *xml, int modules,
 
         chld_node = OS_GetElementsbyNode(xml, node[i]);
 
-        if (chld_node && (strcmp(node[i]->element, osglobal) == 0)) {
-            if (((modules & CGLOBAL) || (modules & CMAIL))
-                    && (Read_Global(chld_node, d1, d2) < 0)) {
+        /* Common configuration */
+		if (chld_node && (strcmp(node[i]->element, oslabels) == 0)) {
```
Comment (TomasTurina) [inline src/config/config.c:69]: Remove extra tabs present in all this file.

File: src/config/config.c [hunk 3]
```diff
@@ -153,51 +146,61 @@ static int read_main_elements(const OS_XML *xml, int modules,
...
 #endif
-        } else if (strcmp(node[i]->element, osgcp) == 0) {
-            if ((modules & CWMODULE) && (Read_GCP(xml, node[i], d1) < 0)) {
+        } else if (chld_node && (strcmp(node[i]->element, task_manager) == 0)) {
+#if !defined(WIN32) && !defined(CLIENT)
+			if ((modules & CWMODULE) && (Read_TaskManager(xml, node[i], d1) < 0)) {
+				goto fail;
+			}
+#else
+			mwarn("%s configuration is only set in the manager.", node[i]->element);
```
Comment (TomasTurina) [inline src/config/config.c:155]: Remove this. It is not necessary the `ifdef`.

File: src/config/config.c [hunk 4]
```diff
@@ -153,51 +146,61 @@ static int read_main_elements(const OS_XML *xml, int modules,
 #else
             mwarn("%s configuration is only set in the manager.", node[i]->element);
```
Comment (TomasTurina) [inline src/config/config.c:147]: Remove this. It is not necessary the `ifdef`.

**Review by TomasTurina — CHANGES_REQUESTED** [review]

File: src/win32/setup-syscheck.c (hunk 1 above)
Comment (cborla) [reply src/win32/setup-syscheck.c]: Done

File: src/shared/debug_op.c [hunk 5]
```diff
@@ -227,58 +227,66 @@ void w_logging_init(){
...
 
-    free(logformat);
-    OS_ClearXML(&xml);
-  }
+    }else{
+
+        parts = OS_StrBreak(',', logformat, 2);
+        char * part;
+        if (parts){
+        for (i=0; parts[i]; i++){
```
[the remark this answers is not in the document]
Comment (cborla) [reply src/shared/debug_op.c]: Done

File: src/shared/debug_op.c [hunk 6]
```diff
@@ -227,58 +227,66 @@ void w_logging_init(){
...
-      free(parts);
+    OS_XML xml;
+    const char * xmlf[] = {"wazuh_config", "logging", "log_format", NULL};
+    char * logformat;
+    char ** parts = NULL;
+    int i;
+
+    pid = (int)getpid();
+#ifdef CLIENT
+    if (OS_ReadXML(chroot_flag ? WAZUHCONF_AGENT : DEFAULTCPATH_AGENT, &xml) < 0){
```
[the remark this answers is not in the document]
Comment (cborla) [reply src/shared/debug_op.c]: Done

File: src/shared/cluster_utils.c [hunk 7]
```diff
@@ -84,7 +84,7 @@ char *get_master_node(void) {
 char *get_node_name(void) {
     OS_XML xml;
     const char * xmlf[] = {"wazuh_config", "cluster", "node_name", NULL};
-    const char *cfgfile = DEFAULTCPATH;
+    const char *cfgfile = DEFAULTCPATH_AGENT;
```
[the remark this answers is not in the document]
Comment (cborla) [reply src/shared/cluster_utils.c]: Done

File: src/shared/cluster_utils.c [hunk 8]
```diff
@@ -63,7 +63,7 @@ int w_is_worker(void) {
 char *get_master_node(void) {
     OS_XML xml;
     const char * xmlf[] = {"wazuh_config", "cluster", "nodes", "node", NULL};
-    const char *cfgfile = DEFAULTCPATH;
+    const char *cfgfile = DEFAULTCPATH_AGENT;
```
[the remark this answers is not in the document]
Comment (cborla) [reply src/shared/cluster_utils.c]: Done

File: src/shared/cluster_utils.c [hunk 9]
```diff
@@ -18,7 +18,7 @@ int w_is_worker(void) {
     const char * xmlf[] = {"wazuh_config", "cluster", NULL};
     const char * xmlf2[] = {"wazuh_config", "cluster", "node_type", NULL};
     const char * xmlf3[] = {"wazuh_config", "cluster", "disabled", NULL};
-    const char *cfgfile = DEFAULTCPATH;
+    const char *cfgfile = DEFAULTCPATH_AGENT;
```
[the remark this answers is not in the document]
Comment (cborla) [reply src/shared/cluster_utils.c]: Done

File: src/config/config.c (hunk 4 above)
Comment (cborla) [reply src/config/config.c]: Done

File: src/config/config.c (hunk 3 above)
Comment (cborla) [reply src/config/config.c]: Done

File: src/config/config.c (hunk 2 above)
Comment (cborla) [reply src/config/config.c]: Done

File: etc/templates/config/README.md [hunk 10]
```diff
@@ -2,7 +2,7 @@
 
 
 
-## Manager 'ossec.conf' file
+## Manager 'manager.conf' file
```
[the remark this answers is not in the document]
Comment (cborla) [reply etc/templates/config/README.md:5]: Done

**Review by TomasTurina — APPROVED** [review]
LGTM!

**Pull request merged**